wgpu-hal/src/lib.rs

```diff
@@ -675,7 +681,7 @@ pub trait Queue: WasmNotSendSync {
 
     /// Submit `command_buffers` for execution on GPU.
     ///
-    /// If `signal_fence` is `Some(fence, value)`, update `fence` to `value`
+    /// If `signal_fence` is `(fence, value)`, update `fence` to `value`
     /// when the operation is complete. See [`Fence`] for details.
```

Comment:
```suggestion
    /// Will update `fence` to `value` when the operation is complete.
    /// See [`Fence`] for details.
```

Reply:
I've incorporated a change similar to this, so I'm going to resolve this.